src/app/tool-editor/tool-info/tool-info.component.ts

```diff
@@ -0,0 +1,167 @@
+    <div class="tool-meta-item">
+      <h4>License</h4>
+      <div>{{model.customProps['sbg:license']}}</div>
+    </div>
+    <div class="tool-meta-item">
+      <h4>Contributors</h4>
+      <ul>
+        <li *ngFor="let contrib of contributors">{{contrib}}</li>
+      </ul>
+    </div>
+  </div>
+  <div class="tool-info-item tool-links">
+    <h3>Links</h3>
+    <ul>
+      <li *ngFor="let link of links"><a href="" (click)="$event.preventDefault(); openWebPage(link.id)">{{link.label}}</a></li>
+    </ul>
+  </div>
+  <div class="tool-info-item tool-details">
+    <ct-tabs-component>
+      <ct-tab-component tabTitle="Inputs">
+        <div *ngIf="inputs.length === 0">
+          No inputs.
+        </div>
+        <table class="table" *ngIf="inputs.length > 0">
+          <tr>
+            <th>ID</th>
+            <th>Label</th>
+            <th>Type</th>
+            <th>Required</th>
+            <th>Prefix</th>
+            <th>Format</th>
+          </tr>
+          <tr *ngFor="let input of inputs">
+            <th scope="row">{{input.id}}</th>
+            <td>{{input.label}}</td>
+            <td>{{input.type | commandParameterType}}</td>
+            <td>{{input.type.isNullable ? 'False' : 'True'}}</td>
+            <td>prefix</td>
+            <td>format</td>
+          </tr>
+        </table>
+      </ct-tab-component>
+      <ct-tab-component tabTitle="App Settings">
+        <div *ngIf="appSettings.length === 0">
+          No settings.
+        </div>
+        <table class="table" *ngIf="appSettings.length > 0">
+            <tr>
+              <th>ID</th>
+              <th>Label</th>
+              <th>Type</th>
+              <th>Required</th>
+              <th>Prefix</th>
+              <th>Format</th>
+            </tr>
+            <tr *ngFor="let input of appSettings">
+              <th scope="row">{{input.id}}</th>
+              <td>{{input.label}}</td>
+              <td>{{input.type | commandParameterType}}</td>
+              <td>{{input.type.isNullable ? 'False' : 'True'}}</td>
+              <td>prefix</td>
+              <td>format</td>
+            </tr>
+        </table>
+      </ct-tab-component>
+      <ct-tab-component tabTitle="Outputs">
+        <div *ngIf="model.outputs.length === 0">
+          No outputs.
+        </div>
+        <table class="table">
+            <tr>
+              <th>ID</th>
+              <th>Label</th>
+              <th>Type</th>
+              <th>Format</th>
+            </tr>
+            <tr *ngFor="let output of model.outputs">
+              <th scope="row">{{output.id}}</th>
+              <td>{{output.label}}</td>
+              <td>{{output.type | commandParameterType}}</td>
+              <td>format</td>
+            </tr>
+        </table>
+      </ct-tab-component>
+    </ct-tabs-component>
+  </div>
+</div>
+`
+})
+export class ToolInfoComponent implements OnChanges {
+    @Input()
+    model: SBDraft2CommandLineToolModel;
+
+    private createdBy: string;
```

Comment:
AoT compilation will fail if private members are used in templates. Make these public.